CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/select/FreeformSelectionPlugin.java

```diff
@@ -62,11 +62,11 @@ public FreeformSelectionPlugin(final boolean isAdd, final boolean isToggle, fina
 
     // From https://stackoverflow.com/questions/11716268/point-in-polygon-algorithm
     private boolean inFreeformPolygons(float xPoint, float yPoint) {
-        int j = -999;
-        int i = -999;
+        int j;
```

Comment:
I think j can be defined within the for loop below (since it isn't being used elsewhere)